recipes/libelfin/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+
+
+class LibelfinConan(ConanFile):
+    name = "libelfin"
+    description = "C++11 library for reading ELF binaries and DWARFv4 debug information"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aclements/libelfin"
+    license = "MIT"
+    topics = ("conan", "elf", "dwarf", "libelfin")
+
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    exports_sources = "patches/*"
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        with tools.chdir(self._source_subfolder):
+            autotools = AutoToolsBuildEnvironment(self)
+            autotools.make()
+            autotools.install(args=["PREFIX=" + os.path.join(self.source_folder, "package")])
+
+    def package(self):
+        build_dir = "package"
+        lib_dir = os.path.join(build_dir, "lib")
+        include_dir = os.path.join(build_dir, "include")
+        # Includes
+        self.copy("*", dst="include", src=include_dir, keep_path=True)
+        # Libraries
+        if self.options.shared:
+            self.copy("*.dylib*", dst="lib", src=lib_dir, keep_path=False, symlinks=True)
+            self.copy("*.so*", dst="lib", src=lib_dir, keep_path=False, symlinks=True)
+            self.copy("*.dll", dst="bin", src=lib_dir, keep_path=False)
+            self.copy("*.dll.a", dst="lib", src=lib_dir, keep_path=False)
+        else:
+            self.copy("*.a", dst="lib", src=lib_dir, keep_path=False)
+        self.copy("*.lib", dst="lib", src=lib_dir, keep_path=False)
+        # Licenses
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
It looks like the makefile also installs a `.pc` files.
Please set the name of the `pkg_config` generator explicitly.

Reply:
Switched to external CMakeLists.txt